src/dbos-runtime/config.ts

```diff
@@ -34,6 +34,7 @@ export interface ConfigFile {
   telemetry?: TelemetryConfig;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   application: any;
+  environmentVariables: Record<string, string>;
```

Comment:
```suggestion
  env: Record<string, string>;
```
`environmentVariables` is too long..